webtool/test_jobs.py

```
@@ -472,3 +472,47 @@ def test_remove_base_entfernt_datei_aus_job_scope():
+def test_request_merkt_unterschiedliche_basen_vor():
+    """Unterschiedliche Dateien (S2, S3) werden bei laufendem S1 beide vorgemerkt und nicht verworfen."""
+    gestartet = []
+    import threading
+    lock = threading.Lock()
+
+    # Job S1 starten
+    jid1, s1 = jobs.start("P_multi", _scope_cmd(["warte"]), cwd=None, kind="correct", base="S1")
+    try:
+        assert s1 is True
+        # Request für S2 (Slot belegt -> vorgemerkt)
+        jid2, s2 = jobs.request("P_multi", _scope_cmd(["S2"]), cwd=None, kind="correct", base="S2")
```

Ruff (0.16.2)

[warning] Unpacked variable `jid2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)